.eslintrc.js

```diff
@@ -16,6 +16,7 @@ module.exports = {
       'env': 'relay',
       'schemaJson': require('./build/schema.json'),
     }],
+    'generator-star-spacing': 0,
```

Comment:
I'm not sure if there is some better way to handle this: https://github.com/babel/babel-eslint/issues/316